vaapi/vaapidisplay_unittest.cpp

```diff
@@ -134,19 +121,6 @@ VAAPI_DISPLAY_TEST(CreateInvalidHandleDRM) {
     EXPECT_FALSE(display.get());
 }
 
-VAAPI_DISPLAY_TEST(CreateInvalidHandleVA) {
-    NativeDisplay native = {0, NATIVE_DISPLAY_VA};
-    DisplayPtr display = VaapiDisplay::create(native);
-
-    EXPECT_FALSE(display.get());
-
-    native.handle = -1;
-    native.type = NATIVE_DISPLAY_VA;
-    display = VaapiDisplay::create(native);
-
-    EXPECT_FALSE(display.get());
-}
-
```

Comment:
It's passed only because it's a special value, 0 and -1, if you send -2 or 1 to create it will failed. Do we really need this test?